### ISSUE TITLE
Nextcloud Will Not Show OnlyOffice In iFrame

### ISSUE THREAD

**Issue by mhfowler** [YunoHost-Apps/onlyoffice_ynh#142]
### Describe the bug

nextcloud will not show onlyoffice document in iframe

### Context

- Hardware: Lenove Thinkcentre 
- YunoHost version: 11.3.0.2 
- I have access to my server: yes
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: I route my internet traffic to my homeserver through a VPN. But HTTPS and other things are working for other apps, just not this one
  - If yes, please explain:
- Using, or trying to install package version/branch: Nextcloud: 29.0.4~ynh2, OnlyOffice 8.1.1~ynh1

### Steps to reproduce

After installing nextcloud (which is working fine), I installed OnlyOffice, and tried to do the integration. I set the secret key and all appeared to work. I can even see the option to create XLSX and other document types in nextcloud now. 

However, when I go to the document, for example curriculum.xlsx, I see an error that my Nextcloud cannot communicate with oo.commoninternet.net (the domain of only office). Maybe this is an iframe issue? How do I resolve this?

If I go to the url of onlyoffice directly, it works fine and has proper https. Both nextcloud and onlyoffice are working independently, but together they are not working because of this error via the screenshot below. 

![Screenshot from 2024-12-06 15-38-33](https://github.com/user-attachments/assets/9dd7663d-0097-4177-9f34-5e2226b00520)

I also tried changing the domain of onlyoffice to be a subdomain of nextcloud and that did not help either (nextcloud at nc.commoninternet.net and onlyoffice at oo.nc.commoninternet.net)

**Comment by mhfowler** [comment]
indeed, when I added the firefox "Ignore X-Frame-Options" extension, and ignored my domains, then it works. 

but still wondering how to make it work for others without the extension?

**Comment by looowizz** [comment]
A bit late but, hm weird. The only thing i would think of is to double check you inputted the the correct domain of your nextcloud instance when installing onlyoffice?

It should then automatically allow Nextcloud to communicate to it due to this line: https://github.com/YunoHost-Apps/onlyoffice_ynh/blob/7f3c7110358e45668f36a0b6b478e979857239b7/conf/nginx.conf#L16
